webpack.development.config.js

```diff
@@ -8,10 +8,49 @@ module.exports = merge(commonConfig, {
     compress: true,
     port: 9000,
     proxy: {
-      '/print/**': {
+      '/print-v2/**': {
         target: {
           host: '127.0.0.1',
           port: 18082
+        },
+        pathRewrite: {
+          '^/print-v2': '/print'
+        }
+      },
+      '/print-v3/**': {
+        target: {
+          host: '127.0.0.1',
+          port: 18083
+        },
+        pathRewrite: {
+          '^/print-v3': '/print'
+        }
+      },
+      '/print/pdf/**': {
```

Comment:
Why is this needed? Shouldn't it be `/print-v2/pdf/**`?

Reply:
This is needed to be able to redirect to the correct print download URL. If called via example (from localhost:9000), the URLs in `info.json` response looks like this:
```
{
  ...
  "printURL": "http://localhost:9000/print/pdf/print.pdf",
  "createURL": "http://localhost:9000/print/pdf/create.json"
}
```
With this additional proxies we ensure that `http://localhost:9000/print/pdf/` will be redirect to `http://localhost:18082/print/pdf/`

The same applies to the next two entries (`/print/status/` and `/print/report/`) in proxy array and is needed for redirects while operating with print servlet v3